modules/OAuth2Tokens/OAuth2Tokens.php

```diff
@@ -42,18 +42,81 @@
     die('Not A Valid Entry Point');
 }
 
-class OAuth2Tokens extends SugarBean {
+/**
+ * Class OAuth2Tokens
+ */
+class OAuth2Tokens extends SugarBean
+{
+    /**
+     * @var string
+     */
     public $table_name = 'oauth2tokens';
+
+    /**
+     * @var string
+     */
     public $object_name = 'OAuth2Tokens';
+
+    /**
+     * @var string
+     */
     public $module_dir = 'OAuth2Tokens';
+
+    /**
+     * @var bool
+     */
     public $disable_row_level_security = true;
+
+    /**
+     * @var
```

Comment:
In the PHPDOC block the type is missing?
```
@var string $tocken_type
@var bool token_is_revoked
@var string access_token_expires
@var string access_token
@var string refresh_token_expires
@var string scopes
@var string state
@var string client
```

Reply:
Thanks